support/keygen-client

```diff
@@ -20,7 +20,7 @@ OPENSSL_VERSION_REGEX_MAJOR_BACKREF="OpenSSL ([0-9]+).*"
 OPENSSL_VERSION_STRING=$(openssl version)
 OPENSSL_VERSION_MAJOR=$(echo "$OPENSSL_VERSION_STRING" | sed -En "s/$OPENSSL_VERSION_REGEX_MAJOR_BACKREF/\1/p")
 
-if [ "$OPENSSL_VERSION_MAJOR" != "1" ]; then
+if [ "$OPENSSL_VERSION_MAJOR" != "1" ] && [ "$OPENSSL_VERSION_MAJOR" != "3" ]; then
```

Comment:
This would be more future proof:

```
if [ "$OPENSSL_VERSION_MAJOR" -lt "1" ]; then
```

That would also include "2", but I think no such version exists.

Reply:
Thanks.
Tested `./keygen-client` with the openssl 3 and this change. Works.